src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -196,6 +201,10 @@ function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
                 subtitle: translate('workflowsPage.delaySubmissionDescription'),
                 switchAccessibilityLabel: translate('workflowsPage.delaySubmissionDescription'),
                 onToggle: (isEnabled: boolean) => {
+                    if (!isEnabled) {
+                        setIsDisableApprovalsConfirmModalOpen(true);
+                        return;
+                    }
```

Comment:
@nkdengineer Why we are adding this here? This does not have relation with approval flow.